example/spec/standalone/redeployment/LambdaRedeploySpec.js

```diff
@@ -1,93 +0,0 @@
-'use strict';
-
-const { LambdaStep } = require('@cumulus/integration-tests/sfnStep');
-const {
-  buildAndStartWorkflow,
-  getLambdaAliases,
-  getLambdaVersions,
-  waitForCompletedExecution
-} = require('@cumulus/integration-tests');
-
-const fs = require('fs-extra');
-const {
-  loadConfig,
-  protectFile,
-  runKes
-} = require('../../helpers/testUtils');
-
-describe('When a workflow is running and a new version of a workflow lambda is deployed', () => {
```

Comment:
I would prefer `xdescribe`ing this test, with a `TODO: reimplement for Terraform` note. It was already significantly refactored in the CUMULUS-1561 branch. If you have issues with the exported helpers being needed feel free to delete the implementations of the suites, but keeping the skeleton would be good.